src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using FluentValidation.Results;
+using Newtonsoft.Json.Linq;
+using NLog;
+using NzbDrone.Common.Cache;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers;
+
+namespace NzbDrone.Core.Applications.LazyLibrarian
+{
+    public class LazyLibrarian : ApplicationBase<LazyLibrarianSettings>
+    {
+        public override string Name => "LazyLibrarian";
+
+        private readonly ILazyLibrarianV3Proxy _lazylibrarianV3Proxy;
```

Comment:
Why is v3 proxy being used?  LL does not have a V3

either drop V3 or replace with V1;  see what is done for Lidarr if V is included

Reply:
Was changed here, but the changes are not appearing upstream. The LazyLibrarian code was copied from mylar which had the same v1 / v3 at the time :-)

Reply:
Unable to use git desktop as there is no Linux version.